自动刷新的文档 (#1682)
cocos-creator/creator-docs: en/editor/preferences/index.md

- **Default Meta**: used to set the default configuration when importing assets within a project. Please refer to the description below for details.
- **Update resources automatically**: automatically refresh resources when returning to the editor from outside (new in v3.1.2). See below for details.

### Update resources automatically (new in v3.1.2)

If this option is enabled, all resources will be automatically checked when returning to Creator, regardless of whether resources have been manipulated outside of Creator. Then when the number of files in the project is too high, or the random read and write speed of the hard disk is too low, it will cause the resource system response lag. For example, if a resource is selected in **Assets** panel, the **Inspector** panel will take a while to show the resource-related properties.

At this point, please disable the automatic refresh feature of Creator. After disabling this feature, if resources are manipulated, then manually click the **Refresh** button at the top right of the **Assets** panel to refresh the resources.

> **Note**: it is not recommended to turn this option off if not experiencing resource system response problems.

Reviewer: ```suggestion
> **Note**: it is not recommended to turn this option off if not experiencing resource system response problems.
```

- **Ignore (Regular)**: use a regular expression and fill in the path to a specific asset file, then that asset will be ignored.
- **Default Meta**: used to set the default configuration when importing assets within a project. Please refer to the description below for details.

### Update resources automatically

> When we do resource manipulation outside of Creator, back in Creator. Creator does a resource check. When the number of files is too high, or the disk random read and write speed is too low, it will cause the resource system response lag. For example, when you click on a resource, the Inspector takes a while to show up.

Reviewer 2: ```suggestion
When performing resource manipulation outside of Creator and bringing the assets into Creator, Creator does a resource check. When the number of files is too high, or the disk random read and write speed is too low, it will cause the resource system response lag. For example, when clicking on a resource, the Inspector takes a while to show up.
```

> At this point, we can turn off the automatic refresh function of Creator, but after turning off this function, we need to click the refresh button in the Assets panel to refresh actively when we have done the resource operation.

Reviewer 2: ```suggestion
At this point, turning off the automatic refresh function of Creator, but after turning off this function, it is necessary to click the refresh button in the Assets panel to refresh actively when the resource operations are done.
```

> So it is not recommended to turn this option off if you are not experiencing resource system response problems.

Reviewer 2: ```suggestion
It is not recommended to turn this option off if not experiencing resource system response problems.
```
